app/policies/application_policy.rb

```diff
@@ -6,6 +6,14 @@ class ApplicationPolicy < ActionPolicy::Base
 
   private
 
+  def administors_collection?(collection)
+    administrator? || depositor? || reviews_collection?(collection) || manages_collection?(collection)
+  end
+
```

Comment:
New common method for both the work_policy and work_version_policy.  If we don't want to have a these methods available to all subclasses, we could move it and the two methods below to a new "work" related superclass